bin/v-add-mail-domain

```diff
@@ -198,6 +198,11 @@ if [ -n "$WEB_SYSTEM" ] || [ -n "$PROXY_SYSTEM" ]; then
 	fi
 fi
 
+# Run extension trigger
+if [ -x "$HESTIA/data/extensions/add-mail-domain.sh" ]; then
+        $HESTIA/data/extensions/add-mail-domain.sh "$domain"
```

Comment:
$HESTIA/data/extensions/add-mail-domain.sh "$user" "$domain"

I don't know for what use cases it might be usefull but lets include $user as well..